affiliates/base/templates/base/home.html

```diff
@@ -17,6 +17,30 @@ <h2 class="site-logo">
 {% endblock %}
 
 {% block content %}
+
+  {% if LANG in settings.FRIENDS_MSG_LOCALES %}
```

Comment:
What's the reasoning for only showing it if it's translated? Wouldn't we want all users to see the message, and we only push it out once it's been localized in our main locales?